README.md

```diff
@@ -144,4 +144,9 @@ for more details:
 **[statically.io](https://statically.io)** (<kbd>demo</kbd> and <kbd>examples</kbd> live environments)
 
 
+<a name="code_coverage">*️⃣ </a> The code coverage doesn't count the code that is only tested through HTML page, like MxGraph. \
+As **puppeteer-to-istanbul** library doesn't support source map, the code coverage based on e2e is on break. \
```

Comment:
```suggestion
As **puppeteer-to-istanbul** library doesn't support the source map, the code coverage based on e2e is broken. \
```

Reply:
⚠️ I would prefer we say that the coverage value is underestimated or under the actual value instead of broken. There is no computation issue. We don't measure coverage on JS code running on the browser.

Reply:
sorry, I forgot to not rebase the branch 😕